.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,41 @@
+## Checklist
+- Contains unit tests ✅ ❌
+- Contains breaking changes ✅ ❌
+- Contains Atlas changes ✅ ❌
+- Compatible with: MX 7️⃣, 8️⃣, 9️⃣
+
+#### Web specific
+- Contains e2e tests ✅ ❌
+- Is accessible ✅ ❌
+- Compatible with Studio ✅ ❌
+- Cross-browser compatible ✅ ❌
```

Comment:
Maybe nice to link to the system requirements docs for Mendix version 7, 8 and 9

Reply:
Lets keep this one in DoD